### PR TITLE
Add more detail

### PR DESCRIPTION
**Pull request by kickoke**
Hey Ian,
Can you review the PR? :)

### CODE REVIEW

File: documentation-and-app-roadmap.md [hunk 1]
```diff
@@ -1,14 +1,32 @@
-# Chronologue Documentation plan
+# Chronologue App and Documentation plan
 
-This document plans the initial set of documentation to enable our users to reach their goals. 
+This document lays down some funamentals regarding the Chronologue app and plans the initial set of documentation. 
+The API is data is stored in this repository and dopleyed with Netlify.
```
Comment (fineon) [inline documentation-and-app-roadmap.md:4]: Did you mean: `dopleyed` to `deploy` ?

File: documentation-and-app-roadmap.md [hunk 2]
```diff
@@ -1,14 +1,32 @@
...
 - Developers
 - Technical writers
 
-## Content outline
+
+## Documentation plan
+
+### Target audience 
+
+Who we need documentation for:
```
Comment (fineon) [inline documentation-and-app-roadmap.md:17]: Did you mean: Who we need `to write` documentation for

**Review by fineon — CHANGES_REQUESTED** [review]
Can you review my comments? Thanks!

File: documentation-and-app-roadmap.md (hunk 1 above)
Comment (kickoke) [inline documentation-and-app-roadmap.md:4]:
```suggestion
The API data is stored in this repository and deployed with Netlify. 
```

File: documentation-and-app-roadmap.md (hunk 2 above)
Comment (kickoke) [inline documentation-and-app-roadmap.md:17]:
```suggestion
Who we need to write documentation for:
```

**Comment by kickoke** [comment]
Thanks for the review :)

**Comment by fineon** [comment]
Looking great. Thanks!

**Pull request merged**